python/tests/unit/core_skills/test_http_skill.py

```diff
@@ -4,8 +4,8 @@
 
 import pytest
 
-from semantic_kernel import Kernel
 from semantic_kernel.core_skills import HttpSkill
+from semantic_kernel.kernel import Kernel
```

Comment:
Why did this change?

Reply:
Ah, I did a blanket change that wasn't necessary for the tests folder. Reverted.